init.php

```diff
@@ -2,66 +2,92 @@
 
 require_once __DIR__ . '/vendor/autoload.php';
 
-use Symfony\Component\Console\Helper\Table;
 use Symfony\Component\Console\Command\Command;
-use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Helper\Table;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\BufferedOutput;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\SingleCommandApplication;
+use Symfony\Component\Console\Style\SymfonyStyle;
 
-CONST VALIDATOR_URL = 'https://validator.prestashop.com';
+const VALIDATOR_URL = 'https://validator.prestashop.com';
 
 (new SingleCommandApplication())
     ->setName('ValidatorDecider') // Optional
     ->setVersion('1.0.0') // Optional
     ->addArgument('github_link', InputArgument::OPTIONAL, 'The github_link')
     ->addArgument('github_branch', InputArgument::OPTIONAL, 'The github_branch')
+    ->addArgument('archive', InputArgument::OPTIONAL, 'The archive')
     ->setCode(function (InputInterface $input, OutputInterface $output) {
 
         $buffer = new BufferedOutput($output->getVerbosity());
         $io = new SymfonyStyle($input, $buffer);
         $client = new \GuzzleHttp\Client([
             'base_uri' => VALIDATOR_URL
         ]);
-        
+
+        $apiKey = null;
+
+        if (getenv('VALIDATOR_API_KEY')) {
+            $apiKey = getenv('VALIDATOR_API_KEY');
+        }
```

Comment:
I guess this is enough to have

```suggestion
        $apiKey = getenv('VALIDATOR_API_KEY');
```

If the env variable does not exist, this will return `false` which is considered as empty by the next check.